### PR TITLE
Form Validation Updates

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6011

*Description of changes:*

* Updated form validation fields

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/common/applicant-info-1.vue [hunk 1]
```diff
@@ -440,12 +437,6 @@ export default class ApplicantInfo1 extends Vue {
   get provinceOptions () {
     return this.$canJurisdictions.map(jurisdiction => ({ value: jurisdiction.value, text: jurisdiction.text }))
   }
-  get provinceStateOptions () {
-    if (this.location === 'IN') {
-      return null
-    }
-    return this.$canJurisdictions
-  }
```
Comment (cameron-freshworks) [inline client/src/components/common/applicant-info-1.vue:448]: Unused code

File: client/src/components/common/applicant-info-1.vue [hunk 2]
```diff
@@ -539,18 +530,15 @@ export default class ApplicantInfo1 extends Vue {
...
   }
   queryAddress (id) {
     newReqModule.getAddressDetails(id)
   }
-  showNextTab () {
-    newReqModule.mutateSubmissionTabComponent('ApplicantInfo2')
-  }
-  showPreviousTab () {
-    newReqModule.mutateSubmissionTabComponent('NamesCapture')
-  }
```
Comment (cameron-freshworks) [inline client/src/components/common/applicant-info-1.vue:553]: Unused code

**Review by severinbeauvais — APPROVED** [review]
Looks OK but you've really changed how each field is functioning, so a couple of us should test this thoroughly in Dev.

File: client/src/components/common/applicant-info-1.vue (hunk 2 above)
Comment (severinbeauvais) [reply client/src/components/common/applicant-info-1.vue:553]: If you were keen, you could also delete this from applicant-info-2 and applicant-info-3. :)

**Comment by cameron-freshworks** [comment]
> Looks OK but you've really changed how each field is functioning, so a couple of us should test this thoroughly in Dev.

Definitely!
Essentially i removed the name fields from the blanket updating method where the validations get reset on any character input. This was causing it to bypass the rules validations until the field was clicked off.
Though i understand why that method does what it does in the first place, so it can allow the auto complete in the address fields to do its job.

File: client/src/components/common/applicant-info-1.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/components/common/applicant-info-1.vue:553]: oh right, will do!

**Pull request merged**